### DISCUSSION TITLE
Connect with certificates data as strings

### DISCUSSION THREAD

**Discussion by riegerben** [psycopg/psycopg2#1676] [Ideas]
HI,
Our code is running on managed clusters without write permissions.
We had alot of problems with the certificate's permissions.
We managed to solve it using temp files.
I want to make a PR for passing the cert's data as strings to the 'connect' func.

Is it a relevant idea?
Is there a PR guide?
Do you even update psycopg2? or just 3...

Thanks

**Comment by dvarrazzo** [comment]
If it's a new feature, it should target psycopg 3. You should describe your idea better.
